systemtest/src/test/java/io/strimzi/systemtest/UserST.java

```diff
@@ -143,6 +145,33 @@ void testBigAmountOfTlsUsers() {
         createBigAmountOfUsers("TLS");
     }
 
+    @Test
+    void testUserWithQuotas() {
```

Comment:
I guess it might be too hard to test the actual quota for example for producer or consumer rate. But should we at least check that the user works?